website_snippet_anchor/templates/snippets.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015, 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). -->
+
+<odoo>
+
+<template id="snippet_options" inherit_id="website.snippets">
+    <xpath expr="//div[@id='snippet_options']" position="inside">
+        <div data-js="anchor"
+             data-selector="section, .carousel">
+            <li data-anchor_ask=""
```

Comment:
There was a `data-only="click"` here before. Leave it as it was if you want to avoid hover events.

Reply:
No, this attribute is not implemented anymore. I'll  use data-no-preview="true" instead.

Reply:
Oh great, I didn't know that :wink: